WordPress/gutenberg
# Fix: Page list: Pages without a title has no link text

## What & Why?

This PR adds two placeholder texts when a page is in a page list block, and the page is missing a title.
The text is `(no title)`, which matches the placeholder for posts, and the placeholder for pages that are not inside a page list.
Without a placeholder title, the link text and button was empty, making the item difficult to see and use. Visually, there was only a blank space and no visible item for a sighted user to click on or navigate to.

Closes https://github.com/WordPress/gutenberg/issues/54959

## How?

The PR adds one placeholder title to the editor, and one to the front.

## Testing Instructions

1. Create three pages that have no titles: Make one of the pages a child of the other (That way submenus can be tested).
2. Add a page list block to the template or content. Confirm that the page list items have visible link texts.
3. Add a navigation block.
4. Add one of the untitled pages to the navigation block.
5. Add a page list block to the navigation block.
6. Confirm that both the page and page list items have visible texts.
7.  Save and go to the front of the site. Confirm that both the page and page list items have visible text.

## Screenshots or screencast 
After: The screenshot shows a navigation block with a  page list inside.
A second page list block is also below (outside) the navigation block.
<img width="659" alt="Page list block with link texts that says (no title)" src="https://github.com/user-attachments/assets/882f4b63-9795-4d3c-a5df-6bb8e0436252">
